lagom10-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/lagom/Env.scala

```diff
@@ -0,0 +1,39 @@
+package com.typesafe.conductr.bundlelib.lagom
+
+import com.typesafe.config.{ Config, ConfigFactory }
+
+import scala.collection.JavaConverters._
+
+/**
+ * Provides functions to set up the Play environment in accordance with what ConductR provides.
+ */
+object Env extends com.typesafe.conductr.bundlelib.scala.Env {
+
+  /**
+   * Provides various Lagom related properties.
+   */
+  def asConfig: Config =
+    ConfigFactory.parseMap(lagomCassandraKeyspace.asJava)
+
+  private final val CassandraKeyspaceNameRegex = """^("[a-zA-Z]{1}[\w]{0,47}"|[a-zA-Z]{1}[\w]{0,47})$"""
+  private def isValidKeyspaceName(name: String): Boolean =
+    name.matches(CassandraKeyspaceNameRegex)
+
+  private def lagomCassandraKeyspace: Map[String, String] =
+    sys.env.get("BUNDLE_NAME").fold(Map.empty[String, String]) { bundleName =>
```

Comment:
Do we already have a val defined for the bundle name in the Scala env? Don't worry if not.